Bookish/Bookish.Web/Models/CatalogueViewModel.cs

```diff
@@ -11,22 +11,51 @@ public class CatalogueViewModel
         public IEnumerable<Book> Books { get; }
         public string SearchTerm { get; }
         public int PageNumber { get; }
+        public int PageItems { get; }
+        public int MaxPages { get; }
 
         public CatalogueViewModel(IEnumerable<Book> books, string searchTerm, int pageNumber)
         {
             Books = books;
             SearchTerm = searchTerm;
-            PageNumber = pageNumber;
+            PageItems = 5;
+            MaxPages = (Books.Count() + PageItems - 1) / PageItems;
+            PageNumber = FixPageNumber(pageNumber);
         }
 
-        public string GetDisabledClass(bool isNext)
+        public int FixPageNumber(int pageNumber)
```

Comment:
Something like `GetValidatedPageNumber` is probably more descriptive? And it can probably be private? 🙂